formula_renames.json

```diff
@@ -86,6 +86,7 @@
   "libcppa": "caf",
   "libmongoclient": "mongo-cxx-driver",
   "libsasl2": "cyrus-sasl",
+  "linux-headers": "linux-headers@4.4",
```

Comment:
Is this typical? I'm thinking not, but I'm not positive.

Reply:
This is a good question.  I think may be that file is for cases where we rename a formula and no longer use the original name as an alias.  Whereas in this case we are changing `linux-headers` to an alias to `linux-headers@4.4`.  

@carlocab @SMillerDev should I be adding something to `formula_renames.json`?  Or is adding the alias sufficient?

Reply:
We usually want an entry in `formula_renames.json` because `brew` needs to know what happened when it sees a user has `linux-headers` installed but no formula named `linux-headers` in the repository.

You could try installing `linux-headers` and then renaming it locally without adding it to `formula_renames.json` locally to see what happens.

Reply:
I tried renaming it in Docker and using `brew doctor` and it does complain that I am missing a formula.  However, adding it to `formula_renames.json` doesn't make that warning go away.  Not sure if there's something else I should be checking instead.

Reply:
@carlocab any thoughts on why adding the formula to `formula_renames.json` didn't fix the issue of `brew doctor` complaining about a missing formula?  If that's expected then I think we can merge this, as it is blocking the bottling of at least two other formulae.

Reply:
I think it doesn't kick in until an upgrade. Try installing `linux-headers`, then rename the formula and add a fake `version "4.4.90"` to your copy of `linux-headers@4.4` and see if `brew upgrade` does the right thing.

Reply:
I did as you suggested and the rename does kick in and works as expected.  I think this makes it clear that we definitely need to add this to `formula_renames.json`.  I'm going to merge this now as that was the only remaining question.